plugins/modules/template_workflow_manager.py

```diff
@@ -1219,7 +1236,7 @@ def get_device_types(self, device_types):
         """
 
         if device_types is None:
-            self.msg = "Mandatory parameter 'device_types' is required."
+            self.msg = "Required parameter 'device_types' is required."
```

Comment:
Instead of saying Required parameter is required.. Can we say like like below?

`    self.msg = "The parameter 'device_types' is required but not provided."
`